templates/common/_base/units/ovs-configuration.service.yaml

```diff
@@ -19,4 +20,4 @@ contents: |
   StandardError=journal+console
 
   [Install]
-  WantedBy=network-online.target
+  WantedBy=multi-user.target
```

Comment:
This one is by far the biggest change, and the one I was aiming towards in https://github.com/openshift/machine-config-operator/pull/3865